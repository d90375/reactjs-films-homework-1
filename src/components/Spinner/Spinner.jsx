import React from 'react';

import styles from './Spinner.scss';

const Spinner = () => (
  <div className={styles.container}>
    <div className={styles.ldsSpinner}>
      <div />
      <div />
      <div />
      <div />
      <div />
      <div />
      <div />
      <div />
      <div />
      <div />
      <div />
      <div />
    </div>
  </div>
);

export default Spinner;
