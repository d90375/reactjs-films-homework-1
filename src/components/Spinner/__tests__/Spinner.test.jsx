import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import Spinner from '../Spinner';

test('Logo renders correctly', () => {
  const renderer = new ShallowRenderer();
  renderer.render(<Spinner />);
  const result = renderer.getRenderOutput();
  expect(result).toMatchSnapshot();
});
