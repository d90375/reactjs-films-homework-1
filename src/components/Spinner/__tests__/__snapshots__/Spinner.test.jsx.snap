// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner renders correctly 1`] = `
<div
  className="container"
>
  <div
    className="ldsSpinner"
  >
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
  </div>
</div>
`;
